en/flight_controller/omnibus_f4_sd.md

```diff
@@ -135,27 +135,41 @@ Here is an example implementation. I used a Spektrum plug to get 3.3v from the D
 ![Omnibus F4 SD Pullup Implementation](../../assets/flight_controller/omnibus_f4_sd/pullup.jpg)
 
 
-## CRSF Crossfire Telemetry
-In addition to the [FrSky Telemetry](../peripherals/frsky_telemetry.md), the Omnibus also supports TBS CRSF Crossfire telemetry.
-It is used to send telemetry data from the flight controller (the vehicles's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+## RC Telemetry
```

Comment:
FYI pushed it down a level because 
- this is RC telemetry, not Wifi telemetry (ie wanted to clarify at heading level)
- We may well want to add a section on FrSky telemetry and now there is a place for that. Reason, I have no idea how you would connect an FrSky receiver for telemetry to this board (?)
- This makes the setup heading a lower level, which in turn differentiates that from the next section on schematics (ie before they looked to be associated).

Reply:
> I have no idea how you would connect an FrSky receiver for telemetry to this board (?)

We're going to need configurable UARTs to make this user-friendly.